plugins/modules/ec2_launch_template_info.py

```diff
@@ -0,0 +1,209 @@
+      - Filter names and values are case sensitive.
+    type: dict
+    default: {}
+
+extends_documentation_fragment:
+  - amazon.aws.common.modules
+  - amazon.aws.region.modules
+  - amazon.aws.boto3
+"""
+
+EXAMPLES = r"""
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+- name: Gather information about a launch template
+  amazon.aws.ec2_launch_template_info:
+    launch_template_ids:
+      - 'lt-01238c059e3466abc'
+
+- name: Gather information launch template using name
+  amazon.aws.ec2_launch_template_info:
+    filters:
+      launch-template-name: my-test-launch-template
+"""
+
+RETURN = r"""
+launch_templates:
+  description: A list of launch templates.
+  returned: always
+  type: list
+  elements: dict
+  contains:
+    launch_template_id:
+      description: The ID of the launch template.
+      type: str
+      returned: always
+    launch_template_name:
+      description: The name of the launch template.
+      type: str
+      returned: always
+    create_time:
+      description: The time launch template was created.
+      type: str
+      returned: always
+    created_by:
+      description: The principal that created the launch template.
+      type: str
+      returned: always
+    default_version_number:
+      description: The version number of the default version of the launch template.
+      type: int
+      returned: always
+    latest_version_number:
+      description: The version number of the latest version of the launch template.
+      type: int
+      returned: always
+    tags:
+      description: A dictionary of tags assigned to image.
+      returned: when AMI is created or already exists
+      type: dict
+      sample: {
+          "Env": "devel",
+          "Name": "nat-server"
+      }
+    versions:
+      description: All available versions of the launch template.
+      type: list
+      elements: dict
+      returned: always
+      contains:
+        launch_template_id:
+          description: The ID of the launch template.
+          type: str
+          returned: always
+        launch_template_name:
+          description: The name of the launch template.
+          type: str
+          returned: always
+        create_time:
+          description: The time the version was created.
+          type: str
+          returned: always
+        created_by:
+          description: The principal that created the version.
+          type: str
+          returned: always
+        default_version:
+          description: Indicates whether the version is the default version.
+          type: bool
+          returned: always
+        version_number:
+          description: The version number.
+          type: int
+          returned: always
+        version_description:
+          description: The description for the version.
+          type: str
+          returned: always
+        launch_template_data:
```

Comment:
should we (are we?) really be returning this as raw Boto3-style data?

Reply:
it is returned as ansible snaked dict like others modules. I have updated it.